inc/DAO/GameDAO.class.php

```diff
@@ -4,5 +4,150 @@ class GameDAO {
 
     private static $db;
     
+    public static function startDb(){
+        self::$db = new PDOService("Game");
+    }
+
+    public static function selectAllProducts(){
+
+        $sql = "SELECT * FROM game";
+
+        self::$db->query($sql);
+
+        self::$db->execute();
+
+        return self::$db->resultSet();
+    }
+
+    public static function selectProductById( int $id ){
+
+        $sql = "SELECT * FROM game WHERE id=:id";
+
+        self::$db->query($sql);
+        
+        self::$db->bind(":id",$id);
+
+        self::$db->execute();
+
+        return self::$db->singleResult();
+    }
+
+    public static function deletProductById( int $id ){
+
+        $sql = "DELETE FROM game WHERE id=:id";
+
+        self::$db->query($sql);
+
+        self::$db->bind(":id",$id);
+
+        self::$db->execute();
+
+        return self::$db->rowCount();
+    }
+
+    public static function insertNewProduct( Game $product ){
```

Comment:
Do you not think that should be more consistent if you write the method name like this:
insertNewGame() ?